### PR TITLE
adding .github folder and templates

### PR DESCRIPTION
**Pull request by ekryski**
@feathersjs/core-team please review and add feedback. We'll iterate in this PR and then can add a script to add this folder to all the other repos as outlined in #435.

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Steps to reproduce
+
+(First please check that this issue is not already solved as [described
+here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))
+
+- [ ] Tell us how you broke things. The more detailed the better.
```
Comment (daffl) [inline .github/issue_template.md]: Maybe less accusing? Something like "Tell us what broke."

Comment (ekryski) [inline .github/issue_template.md]: lol true dat

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+### Actual behavior
+Tell us what happens instead
+
+### System configuration
+**Module versions** (especially the part that's not working):
+
+**NodeJS version**:
+
+**Operating System**:
```
Comment (daffl) [inline .github/issue_template.md]: Should we ask for the client configuration here, too? Browser version and module loader (if applicable)

Comment (ekryski) [inline .github/issue_template.md]: Ya probably. At least browser version and module loader

**Comment by corymsmith** [comment]
Once we agree on this I can sort the script to add it to all repos, have a good idea on that, should be pretty simple using something like https://github.com/github/hub since I already have all the repos in a folder.

**Comment by ekryski** [comment]
@corymsmith That was my thinking as well. The quick and dirty would be if you already have the folders and write access to simply make a script that rips through all of them and copies the folder to each one, adds it to the end of `.npmignore`, commits and pushes straight to master.

**Comment by corymsmith** [comment]
Exactly what I was thinking, though I was thinking of PR but thats probably overkill

**Comment by ekryski** [comment]
For this stuff I think so. It doesn't have any impact on code and it's going to be the same in every repo.

**Comment by corymsmith** [comment]
I've got a script sorted for this so let me know when we're good to go

**Pull request merged**